src/components/app/content/CardsPanel.js

```diff
@@ -30,4 +30,12 @@ export default function CardsPanel(props) {
             </div>
         </div>
     );
-}
\ No newline at end of file
+}
+
+const mapStateToProps = (state) => {
+    return {
+        checkedCardIds: state.cards.checkedCardIds,
+    };
+};
+
+export default connect(mapStateToProps, { removeCards })(CardsPanel);
```

Comment:
Try out react-redux hooks
https://react-redux.js.org/api/hooks
I'm pretty sure you'd love them.